Description: missing array quoting
 https://github.com/pabs3 report with
 shellcheck --severity error wifi-qr
 https://github.com/kokoye2007/wifi-qr/issues/9
 .
 wifi-qr (0.1-2) unstable; urgency=medium
 .
   * current and hidden arrary
Author: kokoye2007 <kokoye2007@gmail.com>

---
Last-Update: 2020-06-17

Index: wifi-qr-0.1/wifi-qr
===================================================================
--- wifi-qr-0.1.orig/wifi-qr
+++ wifi-qr-0.1/wifi-qr
@@ -111,13 +111,13 @@ call_wifi_terminal() {
   IFS=$'\r\n' GLOBIGNORE='*' command eval  'wifilist=($(nmcli  -g NAME,TYPE connection | grep 802-11-wireless | cut -f1 -d":"))'
   OLD_IFS=$IFS
   IFS=$'\n'
-  for l in ${wifilist[@]}; do
+  for l in "${wifilist[@]}"; do
     array[$i]=$l
     i=$(($i + 1))
   done
 
   i=0
-  for m in ${array[@]}; do
+  for m in "${array[@]}"; do
     echo
     echo "$((i++))) $(basename "$m")"
   done
